desk/lib/dm.hoon

```diff
@@ -436,11 +328,13 @@
     =/  ship  (slav %p ship.pole)
     =/  time  (slav %ud time.pole)
     ?.  ?=(%u care)
-      =/  =writ:c  writ:(got ship `@da`time)
+      =/  writ=(may:c writ:c)  writ:(got ship `@da`time)
+      ?:  ?=(%v3 ver)  ``chat-writ-3+!>(writ)
+      ?:  ?=(%| -.writ)  [~ ~]
       ?-  ver
-        %v0  ``writ+!>((old-writ-3 writ))
-        %v1  ``chat-writ-1+!>((old-writ-4 writ))
-        %v2  ``chat-writ-2+!>(writ)
+          %v0  ``writ+!>((v3:writ:v5:cv (v5:writ:v6:cv +.writ)))
+          %v1  ``chat-writ-1+!>((v4:writ:v5:cv (v5:writ:v6:cv +.writ)))
+          %v2  ``chat-writ-2+!>((v5:writ:v6:cv +.writ))
       ==
     ``loob+!>(?~((get ship `@da`time) | &))
```

Comment:
For v0 through v2, this may need to check both whether it's present and whether it's not a tombstone. Otherwise we may give "true" for the `%u` check, but then still produce `[~ ~]` when it gets scried out.